recipes/sdl_mixer/all/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(sdl2_mixer)
```

Comment:
This custom CMakeLists doesn't honor lib name produced by upstream build system, which is `SDL2_mixer`.

Reply:
is it stable and always SDL2_mixer regardless of settings?

Reply:
found it https://github.com/SDL-mirror/SDL_mixer/blob/release-2.0.4/SDL2_mixer.pc.in#L10
I'll change